.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request
+
+on:
+  pull_request:
```

Comment:
Would this also trigger the workflow for a push after opening the PR?

Reply:
Thanks for clarifying! 😃

Reply:
Of course! I knew it worked from experience but I didn't think much about it until I saw your question so I went digging 😅